Resources/skeleton/app/SymfonyRequirements.php

```diff
@@ -425,11 +425,13 @@ public function __construct()
             'Change the permissions of either "<strong>app/logs/</strong>" or  "<strong>var/logs/</strong>" directory so that the web server can write into it.'
         );
 
-        $this->addPhpIniRequirement(
-            'date.timezone', true, false,
-            'date.timezone setting must be set',
-            'Set the "<strong>date.timezone</strong>" setting in php.ini<a href="#phpini">*</a> (like Europe/Paris).'
-        );
+        if (version_compare($installedPhpVersion, '7.0.0', '<')) {
```

Comment:
In Symfony, we usually use the `PHP_VERSION_ID` constant to perform such checks:

```php
if (PHP_VERSION_ID < 70000) {
    // ...
}
```